.env.example

```diff
@@ -14,8 +14,7 @@ KODI_PORT=
 KODI_USERNAME=
 KODI_PASSWORD=
 
-SKILL_APPID=
-SKILL_VERIFY_CERT=
```

Comment:
They are two separate things though.  SKILL_APPID is to verify that the application ID matches what we expect, whereas SKILL_VERIFY_CERT is for verifying the certificate.

I personally want them both enabled at all times, but others might just want one or the other.